flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
since it's a python app, why this limitation?

Reply:
Thanks for the information. i have removed this limitation.